src/containers/ReviewsContainer.js

```diff
@@ -1,17 +1,14 @@
-import React from "react";
-import ReviewForm from "../components/ReviewForm";
-import Reviews from "../components/Reviews";
+import React from 'react';
+import ReviewForm from '../components/ReviewForm';
+import Reviews from '../components/Reviews';
 
-class ReviewsContainer extends React.Component {
-  render() {
-    return (
-      <div>
-        <ReviewForm toy={this.props.toy} />
-        <br />
-        <Reviews reviews={this.props.toy && this.props.toy.reviews} />
-      </div>
-    );
-  }
-}
+const ReviewsContainer = ({ toy }) => {
+  return (
+    <div>
+      <ReviewForm toy={toy} />
+      <Reviews reviews={toy && toy.reviews} />
+    </div>
+  );
+};
```

Comment:
Ditto here about shortcuts if you're only returning from a function:

```
const ReviewsContainer = ({ toy }) => (
  <div>
    <ReviewForm toy={toy} />
    <Reviews reviews={toy && toy.reviews} />
  </div>
};
```

Reply:
I need to nail this into my head! Thank you